### PR TITLE
HL-181

### PR DESCRIPTION
**Pull request by bazmatic**
https://dltx.atlassian.net/browse/HL-181

- Use Bytes16 for nonce, marketId, propositionId
- Can run via local chain by setting env var `NETWORK`
- Fixed typos
- Commented out unused `getRaceResults()`

### CODE REVIEW

**Review by brettvniekerk — CHANGES_REQUESTED** [review]
Can we merge this after the settling refactor is in, because the FE will need to be updated post-refactor to handle bytes16

**Review by OptimusOpus — APPROVED** [review]
LGTM

**Review by pondpiu — APPROVED** [review]

**Review by OptimusOpus — DISMISSED** [review]

**Review by pondpiu — APPROVED** [review]

**Review by pondpiu — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -282,25 +290,45 @@ const signBackMessage = (
...
     timeZone: "Australia/Brisbane"
   });
   moment.suppressDeprecationWarnings = true;
   return moment(today).format(format ?? "YYYY-MM-DD");
-};
+}
 
 const getMeetings = async date => {
+  const url = `https://api.beta.tab.com.au/v1/tab-info-service/racing/dates/${date}/meetings?jurisdiction=QLD&returnOffers=true&returnPromo=false`;
+  console.log(`Fetching meetings ${url}`);
```
Comment (OptimusOpus) [inline src/index.js:328]: Did you mean to leave this in?

File: src/index.js [hunk 2]
```diff
@@ -413,6 +441,11 @@ const getMeetings = async date => {
   return meetings;
 };
 
+app.use((req, res, next) => {
+  console.log(req.url);
```
Comment (OptimusOpus) [inline src/index.js:445]: Did you mean to leave this in?

**Review by OptimusOpus — APPROVED** [review]
Just a few console.logs that I am not sure if you meant to leave. If they are meant to be there then I am happy to approve

**Review by brettvniekerk — APPROVED** [review]

**Pull request merged**